app/reports/mi-report/get-event-order.js

```diff
@@ -0,0 +1,38 @@
+const {
+  PAYMENT_EXTRACTED,
+  PAYMENT_ENRICHED,
+  PAYMENT_PAUSED_DEBT,
+  PAYMENT_DEBT_ATTACHED,
+  PAYMENT_PAUSED_LEDGER,
+  PAYMENT_LEDGER_ASSIGNED,
+  PAYMENT_PAUSED_QUALITY_CHECK,
+  PAYMENT_FAILED_QUALITY_CHECK,
+  PAYMENT_PASSED_QUALITY_CHECK,
+  PAYMENT_PROCESSED,
+  PAYMENT_SUBMITTED,
+  PAYMENT_ACKNOWLEDGED,
+  PAYMENT_SETTLED
+} = require('../../constants/events')
+
+const getEventOrder = (event) => {
+  const events = [
```

Comment:
Should the name of this remain eventsOrder, or is that implied by the name of the file/function?

Reply:
I thought it was clear enough from the name, but if you think it's misleading I'm not precious about it.

Reply:
Yeah, I think rename it, just to not assume anything

Reply:
Renamed :)